### PR TITLE
Add asset pipeline

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-blog/issues/17

### CODE REVIEW

File: bin/asset-pipeline [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+    let css = await fs.readFile(origPath, 'utf8');
+    let replacementsCount = 0;
+    const output = await postcss()
+      .use(
+        postcssURL({
+          url: (asset) => {
+            let assetURL = asset.url;
+
+            // This is a special case to support relative paths in CSS files.
+            if (assetURL.startsWith('../')) {
```
Comment (willdurand) [inline bin/asset-pipeline:300]: 💣

Comment (willdurand) [reply bin/asset-pipeline:300]: I added test coverage, I think it's pretty okay for now. That will depend on the new mockups to implement too.

File: package.json [hunk 2]
```diff
@@ -47,10 +51,25 @@
...
+    "posthtml-postcss": "0.5.0",
+    "posthtml-urls": "1.0.0",
     "prettier": "^2.2.1",
     "pretty-quick": "^3.1.0",
     "prismjs": "^1.23.0",
     "stylelint": "^13.11.0",
-    "stylelint-config-standard": "^20.0.0"
-  }
+    "stylelint-config-standard": "^20.0.0",
+    "svgo": "1.3.2",
```
Comment (muffinresearch) [inline package.json:65]: I don't think this is used (yet)

**Review by muffinresearch — APPROVED** [review]
r+wc

**Pull request merged**